site/src/pages/WorkspaceSettingsPage/WorkspaceParametersPage/WorkspaceParametersPage.tsx

```diff
@@ -87,9 +122,20 @@ export const WorkspaceParametersPageView: FC<
         <ErrorAlert error={submitError} css={{ marginBottom: 48 }} />
       )}
 
+      {workspace.outdated &&
+        workspace.template_require_active_version &&
+        !canChangeVersions && (
+          <Alert severity="warning" css={{ marginBottom: 48 }}>
+            The template for this workspace requires automatic updates. Update
+            the workspace to edit parameters.
+          </Alert>
+        )}
```

Comment:
This should go inside `WorkspaceParametersForm` for a couple of reasons...

- This large, complicated condition is a duplicate of a variable that that component already defines
- As is, we would show this warning even on workspaces that have no parameters. The condition right below this decides whether to show the form or an empty state.